container/lxd/initialisation_test.go

```diff
@@ -469,5 +475,5 @@ func (s *InitialiserSuite) TestBridgeConfigurationWithNewSubnet(c *gc.C) {
 	result, err := bridgeConfiguration(`LXD_IPV4_ADDR=""`)
 	c.Assert(err, gc.IsNil)
 	actualValues := parseLXDBridgeConfigValues(result)
-	c.Assert(expectedValues, gc.DeepEquals, actualValues)
+	c.Assert(actualValues, gc.DeepEquals, expectedValues)
 }
```

Comment:
Do you have any test that shows it discarding a candidate octet because it's in use? (I looked for something with a subnet of 5.) Not sure how hard that is, but it'd be good if it isn't too difficult.